wherehows-web/app/typings/api/datasets/relationships.d.ts

```diff
@@ -15,4 +15,15 @@ type Relationships = Array<IDatasetView>;
  */
 type RelationshipType = IDropDownOption<string>;
 
-export { Relationships, RelationshipType };
+/**
+ * Relationship upstream and downstream api will return Array<IDatasetLineage> displaying
+ * the type of dataset, type of lineage, and actor urn that modified that relationship
+ */
+interface IDatasetLineage {
+  dataset: IDatasetView;
+  type: string;
+  actor: string;
+  modified: string;
```

Comment:
What is `modified`? don't see this in the java Class

Reply:
According to discussions with Mars, modified should exist in the java class but he hasn't added it on the backend side yet. I guess in both cases we can add it as undefined.